<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StageSetting</class>
 <widget class="QWidget" name="StageSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>1338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wBg" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#wBg{background-color: rgb(255, 255, 255);}
</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>4</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="wTitle" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>160</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>160</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(102, 102, 102);</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>American Typewriter</family>
                <pointsize>18</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>設定畫面</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QWidget" name="wBody" native="true">
           <layout class="QGridLayout" name="gridLayout_10">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item row="0" column="0">
             <widget class="QStackedWidget" name="stackedWidget">
              <widget class="QWidget" name="page">
               <layout class="QGridLayout" name="gridLayout_23">
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <property name="verticalSpacing">
                 <number>40</number>
                </property>
                <item row="9" column="0">
                 <widget class="ItemSetting" name="vNetSetting3" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>120</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>400</height>
                   </size>
                  </property>
                  <zorder>verticalSpacer</zorder>
                  <zorder>vNetSetting2</zorder>
                 </widget>
                </item>
                <item row="8" column="0">
                 <widget class="ItemSetting" name="vNetSetting2" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>120</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>400</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QWidget" name="wFreeSetting" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>120</height>
                   </size>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_8">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <item row="0" column="0">
                    <widget class="QWidget" name="w0" native="true">
                     <property name="minimumSize">
                      <size>
                       <width>121</width>
                       <height>40</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>200</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QWidget#w0{ border: 2px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;}</string>
                     </property>
                     <layout class="QGridLayout" name="gridLayout_4">
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <item row="0" column="0" rowspan="2">
                       <widget class="QWidget" name="w0_0" native="true">
                        <property name="minimumSize">
                         <size>
                          <width>0</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>200</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <layout class="QGridLayout" name="gridLayout_7">
                         <property name="leftMargin">
                          <number>5</number>
                         </property>
                         <property name="topMargin">
                          <number>5</number>
                         </property>
                         <property name="rightMargin">
                          <number>5</number>
                         </property>
                         <property name="bottomMargin">
                          <number>5</number>
                         </property>
                         <item row="0" column="0">
                          <widget class="QPushButton" name="btn0_2">
                           <property name="sizePolicy">
                            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                            </sizepolicy>
                           </property>
                           <property name="minimumSize">
                            <size>
                             <width>160</width>
                             <height>0</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>160</width>
                             <height>16777215</height>
                            </size>
                           </property>
                           <property name="styleSheet">
                            <string notr="true"/>
                           </property>
                           <property name="text">
                            <string>自訂</string>
                           </property>
                           <property name="checkable">
                            <bool>true</bool>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </widget>
                      </item>
                      <item row="0" column="1" rowspan="2">
                       <widget class="QWidget" name="w0_1" native="true">
                        <property name="styleSheet">
                         <string notr="true">QWidget#w0_1{    border: 1px solid gray;
    padding: 1px 18px 1px 3px;
    min-width: 6em;}</string>
                        </property>
                        <layout class="QGridLayout" name="gridLayout_11">
                         <property name="topMargin">
                          <number>20</number>
                         </property>
                         <property name="bottomMargin">
                          <number>20</number>
                         </property>
                         <property name="horizontalSpacing">
                          <number>0</number>
                         </property>
                         <item row="0" column="1">
                          <widget class="QLineEdit" name="lineEdit">
                           <property name="sizePolicy">
                            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                            </sizepolicy>
                           </property>
                          </widget>
                         </item>
                         <item row="0" column="2">
                          <widget class="QLabel" name="lb0">
                           <property name="minimumSize">
                            <size>
                             <width>40</width>
                             <height>0</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>40</width>
                             <height>16777215</height>
                            </size>
                           </property>
                           <property name="text">
                            <string>PORT:</string>
                           </property>
                          </widget>
                         </item>
                         <item row="0" column="3">
                          <widget class="QLineEdit" name="lineEdit_2">
                           <property name="sizePolicy">
                            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                             <horstretch>0</horstretch>
                             <verstretch>0</verstretch>
                            </sizepolicy>
                           </property>
                          </widget>
                         </item>
                         <item row="0" column="0">
                          <widget class="QLabel" name="lbTitle0">
                           <property name="minimumSize">
                            <size>
                             <width>40</width>
                             <height>0</height>
                            </size>
                           </property>
                           <property name="maximumSize">
                            <size>
                             <width>40</width>
                             <height>16777215</height>
                            </size>
                           </property>
                           <property name="font">
                            <font>
                             <pointsize>14</pointsize>
                            </font>
                           </property>
                           <property name="text">
                            <string>IP:</string>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="7" column="0">
                 <widget class="ItemSetting" name="vNetSetting1" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>120</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>400</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="11" column="0">
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="6" column="0">
                 <widget class="ItemSetting" name="vNetSetting0" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>121</width>
                    <height>120</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>400</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_2"/>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="wSub" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>90</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>100</height>
            </size>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>其它</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>16</pointsize>
               </font>
              </property>
              <property name="text">
               <string>網路</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemSetting</class>
   <extends>QWidget</extends>
   <header>ItemSetting.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
